src/api/bulidPayTransaction.test.ts

```diff
@@ -0,0 +1,134 @@
+import { base, mainnet } from 'viem/chains';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+import type { Mock } from 'vitest';
+import { PAY_HYDRATE_CHARGE } from '../network/definitions/pay';
+import { sendRequest } from '../network/request';
+import {
+  PAY_INVALID_CHARGE_ERROR_MESSAGE,
+  PAY_UNSUPPORTED_CHAIN_ERROR_MESSAGE,
+  UNCAUGHT_PAY_ERROR_MESSAGE,
+} from '../pay/constants';
+/**
+ * @vitest-environment node
+ */
+import { buildPayTransaction } from './buildPayTransaction';
+import {
+  MOCK_HYDRATE_CHARGE_INVALID_CHARGE_ERROR_RESPONSE,
+  MOCK_HYDRATE_CHARGE_SUCCESS_RESPONSE,
+  MOCK_INVALID_CHARGE_ID,
+  MOCK_VALID_CHARGE_ID,
+  MOCK_VALID_PAYER_ADDRESS,
+} from './fixtures/buildPayTransactionFixtures';
+import type {
+  BuildPayTransactionParams,
+  HydrateChargeAPIParams,
+} from './types';
+
+vi.mock('../network/request');
+
+describe('buildPayTransaction', () => {
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('should return a Pay Transaction', async () => {
+    const mockParams: BuildPayTransactionParams = {
+      address: MOCK_VALID_PAYER_ADDRESS,
+      chainId: base.id,
+      chargeId: MOCK_VALID_CHARGE_ID,
+    };
+    const mockAPIParams: HydrateChargeAPIParams = {
+      sender: MOCK_VALID_PAYER_ADDRESS,
+      chargeId: MOCK_VALID_CHARGE_ID,
+      chainId: base.id,
+    };
+
```

Comment:
nit. let's avoid spaces inside an `it()` so it's easier read many of them at the same time.

But still have one space between `it()`